cpp/unreal_plugins/SpEngine/Source/SpEngine/GameWorldService.h

```diff
@@ -133,7 +320,10 @@ class GameWorldService {
             });
 
         unreal_entry_point_binder->bindFuncUnreal("game_world_service", "find_function_by_name",
-            [this](const uint64_t& uclass, const std::string& name, const EIncludeSuperFlag::Type& include_super_flag) -> uint64_t {
+            [this](const uint64_t& uclass, const std::string& name, const std::map<std::string, std::string>& unreal_obj_strings) -> uint64_t {
```

Comment:
This is super clean. Considering everything this code needs to do, it doesn't get much cleaner than this.